css/pages/index.css

```diff
@@ -375,14 +396,17 @@ div.handles-container {
 }
 
 .video-content > .video-text {
-  margin: 0 auto 40px;
-  font-size: 34px;
+  margin: 45px auto 0px;
+  font-size: 45px;
+  font-weight: 500;
```

Comment:
Can we make it `700` ?? As in the design, it is bold and is same as `700`.

Reply:
I have made it 700..but it didnt looked exactly like figma design. It looked more similar with 500.

Reply:
In the design, it's `600` so can we make that change please ??